### DISCUSSION TITLE
Keep Items in Dock

### DISCUSSION THREAD

**Discussion by amangalampalli** [Homebrew/discussions#2636] [Casks]
I use `brew autoupdate` to automatically update my brew dependencies upon bootup or at specific time intervals. However, some casks are stored in my Dock and when updated disappear from my dock but still appear in the Launchpad. Is there a way to keep these items in my dock even through upgrades/updates?

**Comment by SMillerDev** [comment]
See https://github.com/Homebrew/homebrew-cask/issues/102721
